homeassistant/components/thermostat/__init__.py

```diff
@@ -246,6 +251,11 @@ def is_fan_on(self):
         """Return true if the fan is on."""
         return None
 
+    @property
+    def thermostat_mode(self):
```

Comment:
"operation" refers to what the system is currently *set* to do (heat, 
cool, auto, off), while thermostat_mode refers to what it *is* doing 
(heating, cooling, idle).


On 6/6/2016 10:03 AM, Paulus Schoutsen wrote:
>
> In homeassistant/components/thermostat/__init__.py 
> <https://github.com/home-assistant/home-assistant/pull/2230#discussion_r65906447>:
>
> > @@ -246,6 +251,11 @@ def is_fan_on(self):
> >          """Return true if the fan is on."""
> >          return None
> >
> > +    @property
> > +    def thermostat_mode(self):
>
> How is this different from the |operation| property?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/home-assistant/home-assistant/pull/2230/files/ee924c3c02c0794ca520d1a2ac33d4a673363e2f#r65906447>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AG8DsjnKOiIxY_BCElzcwi_gnEP8VaIlks5qJDamgaJpZM4IuCjU>.
>